app/views/overview/_list-row.html

```diff
@@ -18,4 +18,10 @@
     <bind-service target="row.overlay.state.target"
                   on-close="row.closeOverlayPanel"></bind-service>
   </div>
+  <div ng-if="row.overlay.panelName === 'unbindService'">
+    <unbind-service
+      target="row.overlay.state.target"
+      bindings-by-instance="row.bindings"
```

Comment:
Ok, here will be: 
```html
  <unbind-service
      target="row.overlay.state.target"
      binding="row.bindings"
      on-close="row.closeOverlayPanel"></unbind-service>
```
and in the service instance row template:
```html
    <unbind-service
      target="row.overlay.state.target"
      bindings="row.bindings"
      applications-by-binding="row.state.applicationsByBinding"
      on-close="row.closeOverlayPanel"></unbind-service>
```
So we will have `bindings` and `applicationsByBinding`.